plutus-tx-plugin/executables/profile/Main.hs

```diff
@@ -24,12 +41,117 @@ import qualified PlutusCore.Default        as PLC
 
 import           Control.Lens.Combinators  (_2)
 import           Control.Lens.Getter       (view)
+import           Data.List                 (intercalate)
+import           Data.Maybe                (fromJust)
 import           Data.Proxy                (Proxy (Proxy))
 import           Data.Text                 (Text)
+import           Data.Time.Clock           (NominalDiffTime, UTCTime, diffUTCTime)
 import           Prettyprinter.Internal    (pretty)
 import           Prettyprinter.Render.Text (hPutDoc)
 import           System.IO                 (IOMode (WriteMode), withFile)
 
+data Stacks
```

Comment:
This isn't a _stack_, right? this is more like a _frame_ of a stack, because it's only talking about a particular function. So it's something like `StackFrameTime`?